src/components/EditorSideBar/Blueprints.tsx

```diff
@@ -7,12 +7,20 @@ import { setActiveCourt } from "@/store/reducer/courtSpecDataSlice";
 import { mockTileData } from "../MockData/MockTileData";
 import { useStoreSelector } from "@/store/hooks";
 import { resetAll } from "@/store/reducer/canvasControlSlice";
+import { ICourtData } from "@/interfaces/design";
 
-const Blueprints: React.FC = () => {
+interface Props {
+  fetchedCourtsData: ICourtData[];
+}
+const Blueprints = ({ fetchedCourtsData }: Props) => {
   const dispatch = useDispatch();
   const [activateCourt, setActivateCourt] = useState<string>("");
   const { courtsData } = useStoreSelector((state) => state.courtSpecData);
 
+  const displayedCourt = fetchedCourtsData?.filter((item: ICourtData) => item.isHidden === false);
```

Comment:
it can be written _const displayedCourt = fetchedCourtsData?.filter((item: ICourtData) => !item.isHidden);_

Reply:
Done! thanks